pubspec.yaml

```diff
@@ -16,8 +16,8 @@ dependencies:
 
   receive_sharing_intent:
     git:
-      url: git@github.com:krabbenprgr/receive_sharing_intent.git
-      ref: receive-.txt-v2
+      url: https://github.com/linagora/receive_sharing_intent.git
+      ref: master
```

Comment:
- [ ] demo function in sharing

Reply:
iOS
https://github.com/user-attachments/assets/21dd2231-0baa-48f4-88b1-49fa411d16b1

Reply:
iOS + link + text: 

https://github.com/user-attachments/assets/f2490a10-cbab-42ed-ac66-756de47e619f

Reply:
@nqhhdev can you help me add the demo of android in this case also, please ?

Reply:
DONE !